framework/src/Volo.Abp.AspNetCore.Mvc.UI.Theme.Shared/wwwroot/libs/abp/aspnetcore-mvc-ui-theme-shared/datatables/datatables-extensions.js

```diff
@@ -304,34 +304,22 @@
                 }
             }
 
-            configuration.language = {
-                info: localize("PagerInfo"),
-                infoFiltered: localize("PagerInfoFiltered"),
-                infoEmpty: localize("PagerInfoEmpty"),
-                search: localize("PagerSearch"),
-                processing: localize("ProcessingWithThreeDot"),
-                loadingRecords: localize("LoadingWithThreeDot"),
-                lengthMenu: localize("PagerShowMenuEntries"),
-                emptyTable: localize("NoDataAvailableInDatatable"),
-                paginate: {
-                    first: localize("PagerFirst"),
-                    last: localize("PagerLast"),
-                    previous: localize("PagerPrevious"),
-                    next: localize("PagerNext")
-                }
-            };
+            configuration.language = datatables.defaultConfigurations.language;
 
-            configuration.dom = '<"dataTable_filters"f>rt<"row dataTable_footer"<"col-auto"l><"col-auto"i><"col"p>>';
+            if(configuration.dom){
+                configuration.dom += datatables.defaultConfigurations.dom;
```

Comment:
why do you add the `defaultConfiguration.dom` to the user's configuration dom?

Reply:
datatables defines many doms, for example: `B` for buttons. 
https://datatables.net/reference/option/dom

When developers want to add buttons, they need to customize the dom property, like : https://support.abp.io/QA/Questions/215/How-to-add-PDF-buttons-to-Datatable-in-indexjs